games-emulation/gambatte-jg/gambatte-jg-9999.ebuild

```diff
@@ -15,32 +15,61 @@ if [[ "${PV}" == *9999 ]] ; then
 else
 	SRC_URI="https://gitlab.com/jgemu/${MY_PN}/-/archive/${PV}/${MY_P}.tar.bz2"
 	S="${WORKDIR}/${MY_P}"
-	KEYWORDS="~amd64"
+	KEYWORDS="~amd64 ~arm ~arm64 ~ppc ~ppc64 ~x86"
 fi
 
-LICENSE="BSD GPL-2"
+LICENSE="
+	GPL-2
+	examples? ( 0BSD )
+	jgmodule? ( BSD )
+"
 SLOT="1"
+IUSE="examples +jgmodule shared"
+REQUIRED_USE="|| ( examples jgmodule shared )"
 
 DEPEND="
-	media-libs/jg:1=
-	media-libs/soxr
+	examples? (
```

Comment:
also `media-libs/libglvnd[X]` (for next time)

Reply:
Could you elaborate what you mean? Is there something changing in Gentoo that requires explicitly adding this as a dependency?

Reply:
The binary (I don't remember its name, it was in /usr/bin) links directly with some libGL.so library that is provided by `media-libs/libglvnd`.  I think something pulls it in, probably `media-libs/libsdl2[opengl]`, so it builds fine, but it is flagged by `qa-vdb` as a direct dependency.

Reply:
Ah, thanks for elaborating. I should of thought of this, the `gambatte-example` depends on the `gl.pc` pkgconfig file which I now realize is provided by `media-libs/libglvnd`. However I think this should work without `[X]`, the example only uses libGL, SDL2 and speexdsp.

https://gitlab.com/jgemu/gambatte/-/blob/5fe2c023bde87706ffa5b6dcb1e5c3d981fef99f/src/example/example.cpp